code/game/area/space_station_13_areas.dm

```diff
@@ -255,68 +255,91 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 
 /area/maintenance/aft
 	name = "Aft Maintenance"
-	icon_state = "amaint"
+	icon_state = "aftmaint"
 
 /area/maintenance/aft/upper
 	name = "Upper Aft Maintenance"
+	icon_state = "upperaftmaint"
 
-/area/maintenance/aft/secondary
-	name = "Aft Maintenance"
-	icon_state = "amaint_2"
+/area/maintenance/aft/greater //use greater variants of area definitions for when the station has two different sections of maintenance on the same z-level. Can stand alone without "lesser". This one means that this goes more fore/north than the "lesser" maintenance area.
+	name = "Greater Aft Maintenance"
+	icon_state = "greateraftmaint"
+
+/area/maintenance/aft/lesser //use lesser variants of area definitions for when the station has two different sections of maintenance on the same z-level in conjunction with "greater" (just because it follows better). This one means that this goes more aft/south than the "greater" maintenance area.
+	name = "Lesser Aft Maintenance"
+	icon_state = "lesseraftmaint"
 
 /area/maintenance/central
 	name = "Central Maintenance"
-	icon_state = "maintcentral"
+	icon_state = "centralmaint"
 
-/area/maintenance/central/secondary
-	name = "Central Maintenance"
-	icon_state = "maintcentral"
+/area/maintenance/central/greater
+	name = "Greater Central Maintenance"
+	icon_state = "greatercentralmaint"
+
+/area/maintenance/central/lesser
+	name = "Lesser Central Maintenance"
+	icon_state = "lessercentralmaint"
 
 /area/maintenance/fore
 	name = "Fore Maintenance"
-	icon_state = "fmaint"
+	icon_state = "foremaint"
 
 /area/maintenance/fore/upper
 	name = "Upper Fore Maintenance"
+	icon_state = "upperforemaint"
 
-/area/maintenance/fore/secondary
-	name = "Fore Maintenance"
-	icon_state = "fmaint_2"
+/area/maintenance/fore/greater
+	name = "Greater Fore Maintenance"
+	icon_state = "greaterforemaint"
+
+/area/maintenace/fore/lesser
```

Comment:
Believe this should be /area/maintenance/fore/lesser
![image](https://user-images.githubusercontent.com/66576896/152074382-d192167b-16f3-4a47-9f8c-2bc6cabebb43.png)

Reply:
fixed the typo